ffi/newpassmanagers.cpp

```diff
@@ -357,4 +514,70 @@ LLVMPY_buildFunctionSimplificationPipeline(LLVMPassBuilderRef PBref,
     return llvm::wrap(FPM);
 }
 
+API_EXPORT(void)
+LLVMPY_module_AddModuleDebugInfoPrinterPass(LLVMModulePassManagerRef MPM) {
+    llvm::unwrap(MPM)->addPass(ModuleDebugInfoPrinterPass(llvm::outs()));
+}
+
+API_EXPORT(void)
+LLVMPY_module_AddVerifierPass(LLVMModulePassManagerRef MPM) {
+    llvm::unwrap(MPM)->addPass(VerifierPass());
+}
+
+#define CGSCC_PASS(NAME)                                                       \
+    void LLVMPY_module_Add##NAME(LLVMModulePassManagerRef MPM) {               \
```

Comment:
This is where I think `API_EXPORT` needs adding to enable these functions to be exported from the DLL in Windows:

```suggestion
    API_EXPORT(void) LLVMPY_module_Add##NAME(LLVMModulePassManagerRef MPM) {               \
```